x-pack/plugins/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -71,6 +75,39 @@ export function AlertDetails() {
   const [summaryFields, setSummaryFields] = useState<AlertSummaryField[]>();
   const [alertStatus, setAlertStatus] = useState<AlertStatus>();
 
+  useEffect(() => {
+    if (!alertDetail) {
+      return;
+    }
+
+    const screenDescription = dedent(`The user is looking at an ${
```

Comment:
i18n?

Reply:
@shahzad31 We always use English when communicating with the LLM (think of this as an API call where you wouldn't translate the params either)